### PR TITLE
Change travel admin layout

### PR DESCRIPTION
**Pull request by tomas-stefano**
This is a preparation for the card https://maps.tpondemand.com/entity/10766-create-principles-tab

The stakeholders asked to differentiate travel with rad admin.

This is the first step to do that. I didn't change the nav color
because I don't know if they would want that and I am trying to
keep things super super super simple.

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/views/layouts/travel_insurance_admin.html.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="MAS Retirement Adviser Directory Administration">
```
Comment (malcolmbaig) [inline app/views/layouts/travel_insurance_admin.html.erb]: Reference to retirement needs updating.

File: app/views/layouts/travel_insurance_admin.html.erb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+
```
Comment (malcolmbaig) [inline app/views/layouts/travel_insurance_admin.html.erb]: stray empty line

File: app/views/layouts/travel_insurance_admin.html.erb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+<footer class="footer">
+  <div class="container">
+    <p class="text-muted">Money Advice Service <%= DateTime.now.year %></p>
+  </div>
+</footer>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+<%= javascript_include_tag('admin') %>
+</body>
+</html>
```
Comment (malcolmbaig) [inline app/views/layouts/travel_insurance_admin.html.erb]: I wonder if it's worth DRYing up the commonality between the two admin layouts? Fine to leave it if you think it's too early to bother.

File: app/views/layouts/travel_insurance_admin.html.erb (hunk 2 above)
Comment (tomas-stefano) [reply app/views/layouts/travel_insurance_admin.html.erb]: I just copied from the rad admin layout and both of them had the empty line. Removed.

File: app/views/layouts/travel_insurance_admin.html.erb (hunk 3 above)
Comment (tomas-stefano) [reply app/views/layouts/travel_insurance_admin.html.erb]: It is easy to move so I created a partial and move the admin main content there.

File: app/views/layouts/travel_insurance_admin.html.erb (hunk 1 above)
Comment (tomas-stefano) [reply app/views/layouts/travel_insurance_admin.html.erb]: ♻️

**Comment by tomas-stefano** [comment]
Good comments! Fixed all of them.

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**